src/app/pages/signing/signing.component.html

```diff
@@ -70,20 +70,36 @@
         <!-- Color pickers for text and background -->
         <button class="btn btn-secondary btn-format mr-2 icon-wrapper" type="button">
           <span id="font-color-text">A</span>
-          <label for="colorpicker-font" hidden>Colour picker for font</label>
-          <input [(ngModel)]='signingService.textColor' class="mr-2" id="colorpicker-font" name="fontColorPicker"
+          <label for="color-picker-font" hidden>Colour picker for font</label>
+          <input [(ngModel)]='signingService.textColor' class="mr-2" id="color-picker-font" name="fontColorPicker"
                  type="color" value={{signingService.textColor}}>
           <span [ngStyle]="{'background-color': signingService.textColor}" class="dynamic-color-bar">&nbsp;</span>
         </button>
         <button class="btn btn-secondary btn-format mr-2 icon-wrapper" type="button">
-          <img alt="fill color selector" id="icon" src="../../../assets/images/icons/SigningIcon_fill-color.svg">
-          <label for="colorpicker-background" hidden>Colour picker for background</label>
-          <input [(ngModel)]='signingService.backgroundColor' class="mr-2" id="colorpicker-background"
-                 name="backgroundColorPicker" type="color" value={{signingService.backgroundColor}}>
+          <img alt="fill color selector" id="fill-color-icon"
+               src="../../../assets/images/icons/SigningIcon_fill-color.svg">
+          <label for="color-picker-background" hidden>Colour picker for background</label>
+          <input [(ngModel)]='signingService.backgroundColor' class="mr-2" id="color-picker-background"
+                 name="backgroundcolor-picker" type="color" value={{signingService.backgroundColor}}>
```

Comment:
I don't know if this name gets used anywhere else, so double check
```suggestion
                 name="background-color-picker" type="color" value={{signingService.backgroundColor}}>
```